slf4j-simple/src/main/java/org/slf4j/impl/SimpleLogger.java

```diff
@@ -227,6 +234,16 @@ private static void initConfig(Hashtable config) {
         LOG_FILE = getStringProperty(LOG_FILE_KEY, LOG_FILE);
     }
 
+    private static void overwriteHashtable(Hashtable source, Hashtable destination) {
```

Comment:
How about renaming this method to `overwriteHashtableContents()` or `replaceHashtableContents()` or something like that? Current name is misleading, it could be interpreted as replacing one `Hashtable` instance with another (which is not the case).

Reply:
Renamed to `replaceHashtableContents()`